jarbas/core/tests/test_companies_command.py

```diff
@@ -61,22 +34,24 @@ def test_save_activities(self, update_or_create):
         self.assertEqual(1, len(main))
         self.assertEqual(99, len(secondaries))
 
-    @patch('jarbas.core.management.commands.companies.lzma')
-    @patch('jarbas.core.management.commands.companies.csv.DictReader')
     @patch('jarbas.core.management.commands.companies.Command.save_activities')
-    @patch('jarbas.core.management.commands.companies.Command.serialize')
     @patch('jarbas.core.management.commands.companies.Command.print_count')
     @patch.object(Company.objects, 'create')
-    def test_save_companies(self, create, print_count, serialize, save_activities, rows, lzma):
+    def test_save_companies(self, create, print_count, save_activities):
         self.command.count = 0
-        lzma.return_value = StringIO()
-        rows.return_value = [sample_company_data]
-        serialize.return_value = dict(ahoy=42)
         save_activities.return_value = ([3], [14, 15])
-        self.command.path = 'companies.xz'
+        self.command.path = os.path.join(settings.BASE_DIR, 'jarbas', 'core', 'tests',
+                                         'fixtures', 'companies.xz')
         self.command.save_companies()
-        create.assert_called_with(ahoy=42)
+        create.assert_called_with(additional_address_details=b'', address=b'', city=b'', cnpj=b'',
+                                  email='test@test.com.br', last_updated=b'', latitude=-15.7910966, legal_entity=b'',
+                                  longitude=-47.9508743, name='Test', neighborhood=b'', number=1, opening=None,
+                                  phone=b'', responsible_federative_entity=b'', situation=b'', zip_code=b'',
+                                  situation_date=datetime.date(2005, 9, 24), situation_reason=b'', type='Book',
+                                  special_situation_date=None, state=b'', status=b'', trade_name=b'',
+                                  special_situation=b'')
```

Comment:
What about creating a dictionary and unpacking it to enhance the readability here?

```python

kwargs = {
    'additional_address_details': b'',
    'address': b'',
    …
}
create.assert_called_once_with(**kwargs)
```

Reply:
Okay, I will.